source/fundamentals/bson/utf8-validation.txt

```diff
@@ -25,9 +25,9 @@ processing overhead since it needs to check the data.
 If you *disable* validation, your application avoids the validation processing
 overhead, but cannot guarantee consistent presentation of invalid UTF-8 data.
 
-The driver enables UTF-8 validation by default. It checks documents for any
-characters that are not encoded in a valid UTF-8 format when it transfers data
-between your application and MongoDB.
+The driver enables UTF-8 validation on documents coming from MongoDB by default. 
```

Comment:
Based on the jira ticket, I think it's important to specify that this is for BSON strings. Reworded the sentence in my suggestion a bit to make it flow better with the added information

```suggestion
By default, the driver enables UTF-8 validation on documents from MongoDB servers that contain BSON string values.
```

Reply:
@jordan-smith721 Thanks for the suggestion on flow! I was torn on adding the part about BSON strings since the preceding part of the page doesn't mention them, and (I think) it's implied throughout the page that in this context they are synonymous with documents.